File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// ManualPrecipitationStations SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class ManualPrecipitationStationsMakeContext
{
    public static function call(array $ctxmap, ?ManualPrecipitationStationsContext $basectx): ManualPrecipitationStationsContext
    {
        return new ManualPrecipitationStationsContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# ManualPrecipitationStations SDK utility: make_context
require_relative '../core/context'
module ManualPrecipitationStationsUtilities
  MakeContext = ->(ctxmap, basectx) {
    ManualPrecipitationStationsContext.new(ctxmap, basectx)
  }
end
